### ISSUE TITLE
quran.com fonts are buggy on Chrome on Android

### ISSUE THREAD

**Issue by ahmedre** [quran/quran.com-frontend#548]
![2016-12-30 07 36 47](https://cloud.githubusercontent.com/assets/47771/21561293/bfca0194-ce1f-11e6-9f4c-8c6b3b54b939.png)

works fine on iOS, but on Android, this is what i see (this is Chrome stable). reloading doesn't really help, just when you scroll, you see these types of artifacts.

**Comment by mmahalwy** [comment]
hmm i think i sometimes see that on web but i thought it was just me...

**Comment by ATouhou** [comment]
SAME ISSUE ON MY PHONE (Z5, NEWEST CHROME) COMPLETELY BROKEN FONT ON SOME AYAH, WORDS JUMPED DOWN,   I WAS UNABLE TO USE THE WEBPAGE NOW AND MUST USE ANOTHER ONE (RECITEQURAN.COM)  VERY SAD....

**Comment by mmahalwy** [comment]
Yeah this is a strange problem. Anyone know how to replicate it?

**Comment by arshdkhn1** [comment]
I can replicate it on my chrome. I am trying to debug it but I am gettin `Internal Server Error` with no logs.

**Comment by ATouhou** [comment]
This has not been fixed (atleast not on the production  server)

**Comment by mmahalwy** [comment]
It's not. I don't know what the problem is. I cannot figure it out.

**Comment by ATouhou** [comment]
Tried debugging on my android phone, seems fixable when making the font smaller + lineheight.  
I looks like it is positioning the font wrong sometimes and even at times duplicating.
I tried changing line-height, font-style:normal   text-shadow: 0 0 0     font-smoothing antialiased   and some other things from google nothing helped.

Think the least painfull way to solve this, is to work backwards in commits and see when it was broken...

It really renders the site - not useable - on mobile...
